app/helpers/application_helper.rb

```diff
@@ -1,5 +1,5 @@
 module ApplicationHelper
   def sidebar_current(target)
-    "current "if current_page?(action: target)
+    "current"if current_page?(action: target)
```

Comment:
__[rubocop]__
* Style/SpaceBeforeModifierKeyword: Put a space before the modifier keyword.